source/solvers/navier_stokes_scratch_data.cc

```diff
@@ -386,6 +389,10 @@ NavierStokesScratchData<dim>::calculate_physical_properties()
               const auto surface_tension_model =
                 properties_manager.get_surface_tension(material_interaction_id);
               surface_tension_model->vector_value(fields, surface_tension);
+              // Gather surface tension gradient only if necessary
+              if (!properties_manager.surface_tension_is_constant())
+                surface_tension_model->vector_jacobian(
+                  fields, field::temperature, surface_tension_gradient);
```

Comment:
Are you sure that it is indeed initialized to zero otherwise or if it's not constant you just don't use it at all?

Reply:
It is not used, when it is constant :)